targets/targets.json

```diff
@@ -4672,6 +5022,10 @@
             "network-default-interface-type": "ETHERNET"
         },
         "device_name": "LPC54628J512ET180",
+        "mbed_rom_start": "0x8000000",
```

Comment:
Which memory of `LPC54628J512ET180` is `mbed_rom_start` describing? I can't find a match.